charts/pega/README.md

```diff
@@ -824,6 +824,55 @@ stream:
   # Enter the external secret name below.
   external_secret_name: ""
 ```
+### Stream (externalized Kafka service) test example with bitnami kafka
+
+For testing purpose you can install bitnami kafka using below commands 
+
+```
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm install my-kafka bitnami/kafka
+```
+Once you deploy external kafka you can populate the external kafka bootstrapServer URL and then bootstrap in the values.yaml file used for Pega Deployment for Stream service.
+In the below example "my-kafka" is the release name and "mypega" is the namespace where the kafka is deployed, replace with the correct namespace as per your kafka deployment in kubernetes namespace
```

Comment:
Please update with edits>>
After you deploy a Kafka service in your organization, you can populate the service's Kafka bootstrapServer URL. You configure the bootstrap URL in the Pega values.yaml file in the `Stream (externalized Kafka service) settings`
 section to establish connectivity between Pega and your Kafka cluster as shown in the following example .
In the example, in the `bootstrapServer` parameter, "my-kafka" is the release name and "mypega" is the namespace of the Pega deployment. For your deployment, replace this general example with the specific release name and namespace. When your deployment starts it connects to your Kafka service.

Reply:
submitted with required changes